ISISComputingGroup/EPICS-ioc
# Ticket1405 iris cryo valve

###Description of work

Add IOC for Iris Cryo valve

###To test

Ticket #1405

###Acceptance criteria

1. Switch on IOC.
2. Use PV [IOCNAME]:STATUS to check the status. It should be OPEN or CLOSED
3. Change the status using [IOCNAME]:OPEN:CMD, putting value OPEN or CLOSE
4. Check the status has changed accordingly
5. Check the logic works for the device in simulate mode
6. Check disable records work as expected
7. Repeat for CRYOVALVE_02

#### Code Review


### Functional Tests


### Final steps
